core/src/main/kotlin/com/justai/jaicf/slotfilling/SlotFillingBotContextExt.kt

```diff
@@ -0,0 +1,15 @@
+package com.justai.jaicf.slotfilling
+
+import com.justai.jaicf.context.BotContext
+
+internal const val SLOTFILLING_ACTIVATOR_KEY = "com/justai/jaicf/slotfilling/activator"
+
+internal fun BotContext.setSlotFillingActivator(activatorName: String?) {
+    session[SLOTFILLING_ACTIVATOR_KEY] = activatorName
+}
+
+internal fun BotContext.getSlotFillingActivator() = session[SLOTFILLING_ACTIVATOR_KEY] as? String
+
+internal fun BotContext.setSlotFillingIsFinished() = session.remove(SLOTFILLING_ACTIVATOR_KEY)
```

Comment:
By fact, this function doesn't set anything. I'd rename it to `finishSlotFilling`